tools/evals/baselines/fluency.yaml

```
@@ -446,3 +446,297 @@ samples:
+  - id: fluency-empty-12-none
+    human_score: none
+    inputs:
+      evaluated: "   "
+
+  - id: fluency-whitespace-none
+    human_score: none
+    inputs:
+      evaluated: |
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)